tests/test.sh

```diff
@@ -31,12 +31,30 @@ test_bin() {
   local expect=$1; shift
   local RESULT=-1
   cd $name;
-  bundle exec metadata-json-lint $* metadata.json >/dev/null 2>&1
+  bundle exec metadata-json-lint $* metadata.json >last_output 2>&1
   RESULT=$?
   if [ $RESULT -ne $expect ]; then
-    fail "Failing Test '${name}' (bin)"
+    fail "Failing Test '${name}' (unexpected exit code '${RESULT}' instead of '${expect}') (bin)"
+    echo "    Note: you can examine '${name}/last_output' for any output"
```

Comment:
@rnelson0 I added a hint to the user here that they might get some info from last_output (I say 'might' because the file might be empty)